components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java

```diff
@@ -281,6 +370,13 @@
             "TOKEN_STATE, USER_TYPE, TOKEN_ID, GRANT_TYPE, SUBJECT_IDENTIFIER, ACCESS_TOKEN_HASH, REFRESH_TOKEN_HASH) "
             + "SELECT ?,?,ID,?,?,?," + "?,?,?,?,?,?,?,?,?," + "?,?,? FROM IDN_OAUTH_CONSUMER_APPS WHERE CONSUMER_KEY=?";
 
+    public static final String INSERT_OAUTH2_ACCESS_TOKEN_WITH_IDP_NAME = "INSERT INTO IDN_OAUTH2_ACCESS_TOKEN " +
+            "(ACCESS_TOKEN, REFRESH_TOKEN, CONSUMER_KEY_ID, AUTHZ_USER, TENANT_ID, USER_DOMAIN, TIME_CREATED, " +
+            "REFRESH_TOKEN_TIME_CREATED, VALIDITY_PERIOD, REFRESH_TOKEN_VALIDITY_PERIOD, TOKEN_SCOPE_HASH, " +
+            "TOKEN_STATE, USER_TYPE, TOKEN_ID, GRANT_TYPE, SUBJECT_IDENTIFIER, ACCESS_TOKEN_HASH, REFRESH_TOKEN_HASH," +
+            " IDP_NAME) SELECT ?,?,ID,?,?,?," + "?,?,?,?,?,?,?,?,?," + "?,?,? FROM IDN_OAUTH_CONSUMER_APPS WHERE " +
```

Comment:
Can we remove concatenations in middle

Reply:
This query is not properly changed in this commit as it belongs to access tokens. I have removed the concatenations in my local changes and it will be included in the next PR.

Reply:
ack